docs/reports/peer_testing_2/usability_tasks.md

```diff
@@ -17,19 +40,17 @@ In your pursuit of having an animated text-to-speech experience, you will use ou
     3. Convert your input to a .wav and .txt file
     4. Inform the examiner which avatar you'd like to use: Barb or Boy
 
-2. Let the examiner do the back-end work that would happen if the API was integrated
+2. Let the examiner do the back-end work that would happen if the API was integrated into Kukarella
     a. Send the .wav file, .txt file, language, and avatar selection to the API
     b. Wait for the API to generate the video
 
-3. Watch the video
-
-Repeat this scenario with a different language each time.
+3. Watch the video and provide feedback in the Google Form as you go
 
-### User Scenario 2: Voice Test
+### User Scenario 3: Voice Test
 
 1. On the Kukarella web app
 
-    1. Enter text in a language
+    1. Enter text in a language you're familiar with
```

Comment:
Or any language they want to see! I feel like one theyre familiar with could be helpful, but we also want to test other languages

Reply:
Their feedback about the mouth movements will be most useful if they're familiar with the language, but I agree it would be nice to test other languages. I'm hoping our sample of participants is multi-lingual 🤞